examples/bsoncxx/builder_list.cpp

```diff
@@ -77,7 +78,9 @@ int main(int, char**) {
     //      }
     // }
     //
-    doc = {"Answers", {"Everything", {"The Universe", {"Life", 42}}}};
+    doc = {
+        "Answers",
+        list::document{"Everything", list::document{"The Universe", list::document{"Life", 42}}}};
```

Comment:
I think this needs `extract` calls, based on this error message: https://spruce.mongodb.com/task/cxx_driver_arm_ubuntu1804_compile_and_test_with_shared_libs_patch_77c52a749571029a09f056867fb9df65b866f222_6011a4f95623437878e68f6d_21_01_27_17_38_03?execution=0

Reply:
Done